C_routing.md

```diff
@@ -619,12 +592,18 @@ We have come quite a long way in this guide without talking about one of the fir
 
 Remember in the [Overview Guide](http://www.phoenixframework.org/docs/overview) when we described plugs as being stacked and executable in a pre-determined order, like a pipeline? Now we're going to take a closer look at how these plug stacks work in the router.
 
-Pipelines are simply plugs stacked up together in a specific order and given a name. They allow us to customize behaviors and transformations related to the handling of requests. Phoenix provides default pipelines for common tasks, but it allows us to customize them, and it also allows us to create new pipelines to meet our needs.
+Pipelines are simply plugs stacked up together in a specific order and given a name. They allow us to customize behaviors and transformations related to the handling of requests. Phoenix provides us with some default pipelines for a number of common tasks. In turn we can customize them as well as creating new pipelines to meet our needs.
```

Comment:
Please change "creating" to "create" for subject verb agreement purposes.

"In turn we can customize them as well as create new pipelines to meet our needs."